Description: Don't show "media" in netbook interface as this is unrelevant in
             non meego interface
Bug: https://bugs.launchpad.net/ubuntu/+source/banshee/+bug/602759

Index: banshee-1.7.2/src/Extensions/Banshee.MeeGo/Banshee.MeeGo/MediaPanelContents.cs
===================================================================
--- banshee-1.7.2.orig/src/Extensions/Banshee.MeeGo/Banshee.MeeGo/MediaPanelContents.cs	2010-07-12 17:49:11.107465002 +0200
+++ banshee-1.7.2/src/Extensions/Banshee.MeeGo/Banshee.MeeGo/MediaPanelContents.cs	2010-07-12 18:20:41.131465002 +0200
@@ -70,7 +70,7 @@
             RedrawOnAllocate = true;
             AppPaintable = true;
 
-            BuildHeader ();
+            //BuildHeader ();
             BuildLibrary ();
             BuildNowPlaying ();
             BuildMessageBar ();
